http-service/src/main/java/net/runelite/http/service/item/ItemService.java

```diff
@@ -475,4 +460,30 @@ public void check()
 		}
 	}
 
+	@Scheduled(fixedDelay = 20_000)
+	public void crawlPrices()
+	{
+		if (tradeableItems == null || tradeableItems.length == 0)
+		{
+			return;
+		}
+
+		int idx = random.nextInt(tradeableItems.length);
+		ItemDefinition current = tradeableItems[idx];
+
+		log.debug("Fetching price for {}", current.id);
+
+		fetchPrice(current.id);
+	}
+
+	@Scheduled(fixedDelay = 1_8000_000) // 30 minutes
+	public void reloadItems() throws IOException
+	{
+		List<ItemDefinition> items = cacheService.getItems();
+		tradeableItems = items.stream()
+			.filter(item -> item.isTradeable)
```

Comment:
There are some items that are tradable, but not on the GE (burnt fish comes to mind)

Reply:
I think that our isTradeable is actually isGeTradeable